data/anro/planets.txt

```diff
@@ -0,0 +1,33 @@
+planet Arun
+	landscape land/water5
+	government "Anro"
+	description `This is a stormy world with a lot of water. The air is cool and smells like sea.`
+	spaceport `The Spaceport is small but it has all it needs.`
+
+planet Airum
+	landscape land/desert0
+	government "Anro"
+	description `This is the world where the Anro originally came from. There are large flat deserts and there is as good as no water around here. Big cities are standing out everywhere and a peacefull feeling lays in the air.`
```

Comment:
```suggestion
	description `This is the world where the Anro originally came from. There are large flat deserts as far as the eye can see, and not a drop of water save the occasional mirage. Only the massive cities stand out within the ocean of sand, standing tall in defiance of the great pale sea.`
```

Reply:
Not going to do my usual proofreading run anytime soon, so that's all you're getting for now.